tests/pytorch/fpga/vgg_conv_optimization_example.py

```diff
@@ -0,0 +1,421 @@
+#!/usr/bin/env python3
+
+# ----------------------------------------
+# VGG-16 Optimization Example
+# ----------------------------------------
+from dace.transformation.interstate import FPGATransformSDFG, InlineSDFG
+from daceml.transformation import InputToConstant
+
+import time
+import math
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import json
+import os
+
+import numpy as np
+
+import daceml.onnx as donnx
+import dace
+from daceml.pytorch import DaceModule, dace_module
+import copy
+
+from daceml.util import utils
+from dace.transformation.dataflow import streaming_memory as sm
+from dace.transformation.dataflow import PruneConnectors
+from dace.transformation.interstate import InlineSDFG
+from dace.sdfg import SDFG, SDFGState, trace_nested_access
+from dace import config, data as dt, dtypes, Memlet, symbolic
+from dace.sdfg import nodes as nd
+
+from dace.transformation.optimizer import Optimizer
+from dace.transformation.dataflow import streaming_memory as sm
+
+import argparse
+
+# Random numbers
+import random
+
+torch.manual_seed(0)
+random.seed(0)
+np.random.seed(0)
+
+# ----------------------------------------
+# Node Mappings
```

Comment:
Could you please explain what are these mappings?

Reply:
Added an explanation